src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -0,0 +1,147 @@
+package nextstep.subway.line.domain;
+
+import nextstep.subway.line.exception.InvalidDataException;
+import nextstep.subway.station.domain.Station;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+@Embeddable
+public class Sections {
+
+    private static final String ALREADY_EXITS_SECTION = "이미 등록된 구간 입니다.";
+    private static final String CAN_NOT_INSERT_INVALID_SECTION = "등록할 수 없는 구간 입니다.";
+    private static final String CAN_NOT_DELETE_LAST_SECTION_IN_LINE = "노선에 구간이 하나 뿐인 경우에는 삭제할 수 없습니다.";
+
+    @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST, CascadeType.MERGE}, orphanRemoval = true)
+    List<Section> sections = new ArrayList<>();
```

Comment:
```suggestion
    private List<Section> sections = new ArrayList<>();
```

값에 접근제한자가 없는데 혹시 의도한 내용이실까요?

Reply:
앗 이 부분은 실수로 빠뜨린 것 같습니다. 😂
꼼꼼하게 피드백 해주셔서 감사합니다. 수정해서 반영했습니다!